forwards_paper.tex

```diff
@@ -1054,24 +1040,28 @@ \section*{Discussion}
 %%%%%%%%%%%%%%
 \section*{Methods}
 
-\paragraph{Requirements.}
-The \cpp{} implementation uses several packages installed via conda,
-except for \msprime{}, which must be installed from GitHub.
+We implemented simulations and the connection to \msprime{} in \cpp{}, using \fwdpp{} library functions and
+interface code using a continuum-sites model for both mutation
+and recombination. Simulations were run using \fwdpy{} (version 0.13.a0), a Python package based
+on \fwdpp{} (version 0.5.7).
+Code for all simulations and figures is available at \url{https://github.com/petrelharp/ftprime_ms}.
+These made use of the GNU Scientific Library \citep[version 1.16][]{galassi2018scientific},
+pybind11 \citep[version 2.2.1][]{pybind11},
+and GCC (version 4.8.5).
+We ran all benchmarks on an Ubuntu Linux (version 16.04) system with two 2.6 GHz Intel E5-2650 CPU with
+hyperthreading enabled.
+We ran one simulation at a time and the machine was under minimal load otherwise.
+We used GNU parallel \citep{Tange2011a} to kill any simulation that did not finish within 72 hours,
+and the Linux \texttt{time} command to record run time and peak memory usage of each replicate.
+%% Our simulation scripts separately recorded the time spent in various steps of each run.
 
-\begin{enumerate}
-    \item \fwdpy{} 0.13.a0
-    \item \fwdpp{} 0.5.7
-    \item GNU Scientific Library 1.16
-    \item pybind11 2.2.1
-    \item GCC 4.8.5
-    \item \msprime{} 0.5.0 \jda{Changed these but the actual sims were not run
-		    with this version...}
-\end{enumerate}
 
 %%%%%%%%%%%%%%%%%%%%%%
 \section*{Acknowledgements}
-Thanks to Brad Shaffer and Evan McCartney--Melstad for useful discussions.
-Funding for this project came from XXX
+Thanks to Jared Galloway, Brad Shaffer, and Evan McCartney--Melstad for useful discussions.
+Work on this project was supported by funding from 
+the Sloan Foundation and the NSF (under DBI-1262645) to PR;
+and XXX.
```

Comment:
@jeromekelleher @ashander @molpopgen any funding you want to acknowledge?

Reply:
USFWS but not sure of the contract number. Will email